scripts/prepdocslib/htmlparser.py

```diff
@@ -0,0 +1,41 @@
+import re
+from typing import IO, AsyncGenerator
+
+from bs4 import BeautifulSoup
+
+from .page import Page
+from .parser import Parser
+
+
+def cleanup_data(data: str) -> str:
+    """Cleans up the given content using regexes
+    Args:
+        data: (str): The data to clean up.
+    Returns:
+        str: The cleaned up data.
+    """
+    output = re.sub(r"\n{2,}", "\n", data)
```

Comment:
Can you comment these, since humans are bad at grokking regexes?

Reply:
yes, of course